transcendence-app/src/game/game.service.ts

```diff
@@ -25,6 +25,7 @@ type ClientId = string;
 const FPS = 30;
 const DELTA_TIME = 1 / FPS;
 const MAX_PAUSED_TIME_MS = 30 * 1000; // 30 seconds
+const MAX_SCORE = 10;
```

Comment:
I think eventually we should try to pass all of these constant as a configuration object, and pull from the configuration, so that we can, you know, have powerups and such, as required by the subject. The whole "constants" concept is highly inflexible for our use case.

Out of scope, I'll admit.

Reply:
:+1: Let's track this in #506